code/game/objects/structures/signs.dm

```diff
@@ -516,4 +511,60 @@
 
 /obj/structure/sign/department/prison
 	name = "PRISON"
-	icon_state = "prison"
\ No newline at end of file
+	icon_state = "prison"
+
+//Eris factions
+
+/obj/structure/sign/faction
+	name = "faction sign"
+	desc = "Faction sign of some sort."
+
+/obj/structure/sign/faction/ironhammer
+	name = "Ironhammer Security"
+	desc = "Sign depicts the symbolic of Ironhammer Security, the largest security provider within Trade Union of Hansa."
+	icon_state = "ironhammer"
+
+/obj/structure/sign/faction/alliance
+	name = "Mankind Alliance"
+	desc = "Alliance's all-seeing eye, logo of now fallen empire. Once they were owning all the Milky Way. Now it's just dust, forgotten derelicts and automated ships."
+	icon_state = "alliance"
+
+/obj/structure/sign/faction/alliance_old
+	name = "Mankind Alliance"
+	desc = "Alliance's all-seeing eye, logo of now fallen empire. Once they were owning all the Milky Way. Now it's just dust, forgotten derelicts and automated ships."
+	icon_state = "alliance-old"
+
+/obj/structure/sign/faction/frozenstar
+	name = "Frozen Star"
+	desc = "Hanza most popular weapon manufacturer."
```

Comment:
"The most popular Hanza's weapon manufacturer." or "The most popular weapon manufacturer within Hanza."